osu.Game/Screens/OsuScreen.cs

```diff
@@ -128,5 +147,33 @@ protected override bool OnExiting(Screen next)
             Beatmap.UnbindAll();
             return false;
         }
+
+        private void logoSetup(bool resuming) => LogoSetup(logo, resuming);

```

Comment:
What's the point of this? Are you missing a `logo.ClearTransforms();` here or is this indirection here expecting this in the future?